builds/misc/images.yaml

```diff
@@ -2,439 +2,9 @@ trigger:
-        parameters:
-          name: Direct Method Receiver
-          imageName: azureiotedge-direct-method-receiver
-          project: DirectMethodReceiver
-
-      # Metrics Validator
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Metrics Validator
-          imageName: azureiotedge-metrics-validator
-          project: MetricsValidator
-
-      # Number Logger
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Number Logger
-          imageName: azureiotedge-number-logger
-          project: NumberLogger
-
-     # Module Restarter
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Module Restarter
-          imageName: azureiotedge-module-restarter
-          project: ModuleRestarter
-
-     # Twin Tester
-      - template: templates/image-linux.yaml
-        parameters:
-          name: TwinTester
-          imageName: azureiotedge-twin-tester
-          project: TwinTester
-
-     # Relayer
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Relayer
-          imageName: azureiotedge-relayer
-          project: Relayer
-
-    # Network Controller
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Network Controller
-          imageName: azureiotedge-network-controller
-          project: NetworkController
-
-     # TestResultCoordinator
-      - template: templates/image-linux.yaml
-        parameters:
-          name: TestResultCoordinator
-          imageName: azureiotedge-test-result-coordinator
-          project: TestResultCoordinator
-
-     # Metrics Collector (test-only)
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Metrics Collector (test-only)
-          imageName: azureiotedge-test-metrics-collector
-          project: MetricsCollector
-
-     # Deployment Tester
-      - template: templates/image-linux.yaml
-        parameters:
-          name: DeploymentTester
-          imageName: azureiotedge-deployment-tester
-          project: DeploymentTester
-
-     # EdgeHub Restart Tester
-      - template: templates/image-linux.yaml
-        parameters:
-          name: EdgeHubRestartTester
-          imageName: azureiotedge-edgehub-restart-tester
-          project: EdgeHubRestartTester
-
-     # Cloud To Device Message Tester
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Cloud To Device Message Tester
-          imageName: azureiotedge-c2dmessage-tester
-          project: CloudToDeviceMessageTester
-
-     # Metrics Collector (customer-facing)
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Metrics Collector (customer-facing)
-          imageName: azureiotedge-metrics-collector
-          project: Microsoft.Azure.Devices.Edge.Azure.Monitor
-
-################################################################################
```

Comment:
We'll need to bring back deleted lines 189-211 here. Jobs with "templates/image-linux-rust-jobs.yaml"